### ISSUE TITLE
Text in navigation footer (previous, next) in not translated

### ISSUE THREAD

**Issue by pfelber** [executablebooks/sphinx-book-theme#359]
### Describe the problem

Hi,

The new navigation footer now has "PREVIOUS" and "NEXT" labels. These labels seem to be hardcoded in the css and are not translated when changing the language of the book. While they can be overridden in the CSS, it might be good to have them generated directly in the book's language (I am using "language: fr").

(As a side note, I preferred the old style with "<<", ">>" and the smaller font.)

Thanks,

Pascal

### Steps to reproduce

This problem can be seen in any generated book when setting the language to anything different from English.

### Versions of your packages

```
❯ jupyter-book --version
Jupyter Book      : 0.11.2
External ToC      : 0.2.2
MyST-Parser       : 0.13.7
MyST-NB           : 0.12.3
Sphinx Book Theme : 0.1.1
Jupyter-Cache     : 0.4.2
NbClient          : 0.5.3
```

# Tasks

- [x] Look into the translation function from https://github.com/executablebooks/sphinx-book-theme/pull/214 to re-use that function
- [ ] Confirm that the translation works as expected

**Comment by chrisjsewell** [comment]
Heya, yeh these navigations are inherited from pydata-sphinx-theme:https://github.com/pydata/pydata-sphinx-theme/blob/f0834772d24c962b7d11643114db88615c43674e/pydata_sphinx_theme/layout.html#L4, and so are not currently handled by the translation functionality included in sphinx-book-theme

So either pydata needs to be "fixed" there (cc @jorisvandenbossche) or we need to override that macro

**Comment by choldgraf** [comment]
Ah this was actually a recent change in the CSS to add "previous" and "next" (it was inspired from the Furo docs). This makes me think we might want to change the implementation as I doubt CSS can be translated unless we turn it into a `_t` template, which seems sub-optimal 🤔

**Comment by chrisjsewell** [comment]
Ah ok yeh no that needs to go in the html templates, so it can be translated

**Comment by chrisjsewell** [comment]
@choldgraf did you check that #374 actually works, because I don't think it will; you are using sphinx's internal translation function which does not include PREVIOUS/NEXT, rather than using the translation function I set up in #214

**Comment by chrisjsewell** [comment]
testing in #381

**Comment by choldgraf** [comment]
ahhh in that case you are right - I didn't realize we needed a separate setup for this. I will keep this on my plate in that case and try to get to it when I have time.

**Comment by choldgraf** [comment]
I can confirm that the translations do not work. I tried using the `translate` function and looked at the [README docs of the translate folder](https://github.com/executablebooks/sphinx-book-theme/blob/master/sphinx_book_theme/translations/README.md) which seems like it would work, with a few confusion points.

@chrisjsewell what languages did you select when you generated the translations? And after that you just put it in a specific folder and then ran `python sphinx_book_theme/translations/_convert.py`? I think we need to have a "step by step" guide for adding a new translation to the theme so that it's straightforward to do this. I can try to write this but I need to know the steps first haha.

**Comment by chrisjsewell** [comment]
> what languages did you select when you generated the translations?

all of them

> nd after that you just put it in a specific folder and then ran python sphinx_book_theme/translations/_convert.py

yep should be as easy as that

**Comment by choldgraf** [comment]
I spent a while trying to figure out the translation machinery and was not able to do so. I don't have time to work on this now so instead I've opened up an issue about documenting this infrastructure, and added in my steps that I took in that issue in case others have time to take a look: https://github.com/executablebooks/sphinx-book-theme/issues/382

I'll leave this issue open in the meantime, though removing myself from assignment because I'm not sure when I'll have a moment to work on coding stuff. We might also consider reverting back to just using `->` instead of english words, so that we don't have to do the translation stuff.
